torch/_inductor/async_compile.py

```diff
@@ -212,6 +212,11 @@ def triton(self, kernel_name: str, source_code: str, device_str: str = "cuda"):
         _compile_start()
         _set_triton_ptxas_path()
 
+        if os.environ.get("TRITON_INTERPRET", "0") == "1":
```

Comment:
Might be better to be in a config so people can see it as an option.

Reply:
You need it to be set as an envvar otherwise triton wont return the interpreted function. So setting the config value wouldnt be enough wed need to thread it to triton as well.. going to leave like this for now